tests/v2/actions/workloads/pods/pods.go

```diff
@@ -1,14 +1,31 @@
 package pods
 
 import (
+	"context"
+	"errors"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/rancher/rancher/tests/v2/actions/kubeapi/workloads/deployments"
+	"github.com/rancher/shepherd/clients/rancher"
+	v1 "github.com/rancher/shepherd/clients/rancher/v1"
+	"github.com/rancher/shepherd/extensions/defaults"
+	"github.com/rancher/shepherd/extensions/kubeconfig"
 	"github.com/rancher/shepherd/extensions/workloads"
 	namegen "github.com/rancher/shepherd/pkg/namegenerator"
+	"github.com/rancher/shepherd/pkg/wait"
+	appv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kwait "k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/apimachinery/pkg/watch"
 )
 
 const (
-	timeFormat = "2006/01/02 15:04:05"
-	imageName  = "nginx"
+	timeFormat   = "2006/01/02 15:04:05"
+	imageName    = "nginx"
+	podSteveType = "pod"
```

Comment:
missing from other PRs: 
```
historyHeaderLength = 2
	revisionNumberIndex = 0
	historyHeader       = "REVISION  CHANGE-CAUSE"
	revisionsIndex      = 1
	revisionAnnotation  = "deployment.kubernetes.io/revision"
```

Reply:
It is not missing, it is in [deployment.go file](https://github.com/rancher/rancher/pull/46875/files#diff-b6843bc14cef727d6cded1a32365e9be967b17b84a7add1b3313d3d48efb4e45R26)